scripts/BuildPackagesFromSigningDirectory.ps1

```diff
@@ -0,0 +1,57 @@
+<#
+.SYNOPSIS
+    Package the SARIF SDK from a custom signing directory.
+.DESCRIPTION
+    Builds the SARIF SDK NuGet Packages from a custom signing directory after
+    they have been signed.
+.PARAMETER Configuration
+    The build configuration: Release or Debug. Default=Release
+#>
+
+[CmdletBinding()]
+param(
+    [string]
+    [ValidateSet("Debug", "Release")]
+    $Configuration="Release"
+)
+
+Import-Module -Force $PSScriptRoot\ScriptUtilities.psm1
+Import-Module -Force $PSScriptRoot\Projects.psm1
+
+# Copy signed binaries back into the normal directory structure.
+function CopyFromSigningDirectory {
+    Write-Information "Copying files to signing directory..."
+    $SigningDirectory = "$BinRoot\Signing"
+
+    foreach ($project in $Projects.NewProduct) {
+        $projectBinDirectory = "$BinRoot\${Platform}_$Configuration\$project\"
```

Comment:
>BinRoot\${Platform}_$Configuration\$project [](start = 33, length = 43)

If this string expression occurs more than once, export a function `Get-ProjectBinDirectory($project, $configuration)` from `ScriptUtilities.psm1`.